File: binarydigits.rb
# http://rosettacode.org/wiki/Binary_digits
start_time = Time.now
test_array = [5,50,9000]

def base_ten_to_binary(number)
  return number.to_s(2)
end

test_array.each do |number|
  puts base_ten_to_binary(number)
end

puts ((Time.now-start_time).to_f).to_s + "s"

File: binary_digits.swift
// http://rosettacode.org/wiki/Binary_digits

func decimalToBinary(n:Int) -> Int {
  if n == 0 {
    return 0
  } else {
    return n % 2 + 10 * decimalToBinary(n/2)
  }
}

println(decimalToBinary(115))